** Temporary build fixes were applied.
   Provided by: Romain Francoise <rfrancoise@debian.org>
---
 lisp/emacs-lisp/bytecomp.el |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/lisp/emacs-lisp/bytecomp.el
+++ b/lisp/emacs-lisp/bytecomp.el
@@ -1935,7 +1935,8 @@
 	   (stringp (nth 3 form)))
       (byte-compile-output-docform nil nil '("\n(" 3 ")") form nil
 				   (memq (car form)
-					 '(autoload custom-declare-variable)))
+					 '(defvaralias autoload
+					   custom-declare-variable)))
     (let ((print-escape-newlines t)
 	  (print-length nil)
 	  (print-level nil)
